pkgs/top-level/python-packages.nix

```diff
@@ -4256,6 +4256,10 @@ let
     };
   };
 
+  # py3k disabled, see https://travis-ci.org/NixOS/nixpkgs/builds/48759067
+  graph-tool = if isPy3k then null
```

Comment:
From what I understand using `disabled` requires `buildPythonPackage` and I didn't use that